src/openvic-simulation/military/LeaderTrait.cpp

```diff
@@ -1,6 +1,7 @@
 #include "LeaderTrait.hpp"
 
 #include "openvic-simulation/modifier/ModifierManager.hpp"
+#include "openvic-simulation/dataloader/NodeTools.hpp"
```

Comment:
```suggestion
#include "openvic-simulation/dataloader/NodeTools.hpp"
#include "openvic-simulation/modifier/ModifierManager.hpp"
```
Include formatting (alphabetical order)